src/ceph-volume/ceph_volume/devices/lvm/common.py

```diff
@@ -48,6 +48,16 @@ def common_parser(prog, description):
         '--osd-fsid',
         help='Reuse an existing OSD fsid',
     )
+    parser.add_argument(
+        '--block.db',
+        dest='block_db',
+        help='(bluestore) Path to bluestore block.db logical volume or device',
```

Comment:
This needs to be a partition, not a raw device. Right?

Reply:
A raw device. No partition needed. We could accept a partition though,  but there is no need to.